docs/integrations/gitops/custom_policy.md

```diff
@@ -0,0 +1,145 @@
+---
+sidebar_position: 3
+title: Write Custom Policies
+---
+
+While Permit.io UI is a good fit for almost every policy configuration need, sometimes developers need to extend their policy configuration to something nonexistent.
+Since Permit.io cloud configuration generates Policy as Code for each user configuration, extending it by writing custom policy code yourself is easy.
+
+This document demonstrates an example of a use-case that is not supported via Permit.io UI but easily possible through Policy as Code extension to the Permit.io UI configuration.
+
+## Deny All Rule
+When a policy is configured in Permit.io UI, whether it’s RBAC, ABAC, or ReBAC, the policy generated is `Allow` policy. This means that if some of the rules allow the user to perform the action, the `check` function will return `true`.
```

Comment:
When a policy is configured in Permit.io's UI, whether it's an RBAC, ABAC, or ReBAC policy, the generated policy rules are Allow rules. This means that if any of the rules allow the user to perform the action, the check function will return true.